src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: $day-padding;
+
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid #000;
+    justify-content: center;
+    align-items: center;
+    width: 100px;
+    height: 100px;
+    transition-duration: 0.5s;
+    display: flex;
+
+    &::before {
+      content: '';
+
+      font-size: $day-font-size;
+      font-family: $day-font-family;
+    }
```

Comment:
The `&::before` pseudo-element should not contain `font-size` and `font-family` properties, as they are not related to content generation. These properties should be applied to the `&__day` element directly.